modules/geo-layers/src/tile-layer/tileset-2d.js

```diff
@@ -161,6 +167,9 @@ export default class Tileset2D {
         changed = true;
         tile.isVisible = isVisible;
       }
+
+      // isSelected used in request scheduler
+      tile.isSelected = Boolean(tile.state === TILE_STATE_SELECTED);
```

Comment:
`tile.state === TILE_STATE_SELECTED` is already boolean

Reply:
Oh I see it was used above because `tile.state & 1` is a number, not a boolean.